### PR TITLE
DOC: Updating docs for the regression summary

### PR DESCRIPTION
**Pull request by mortonjt**
This is just an update of the regression summaries building off of @lisa55asil 's edits

### CODE REVIEW

File: q2_gneiss/regression/_regression.py [hunk 1]
```diff
@@ -78,9 +96,15 @@ def lme_regression(output_dir: str,
...
-        'formula': 'Statistical formula specifying the statistical model.',
+        'formula': 'Statistical formula specifying the statistical model.'
+                   'In other words, a list of the metadata categories that '
+                   'will be used in the linear mixed effect model, '
+                   'typically separated by "+". For more information see '
+                   'https://patsy.readthedocs.io/en/latest/API-reference.html',
         'metadata': ('Metadata information that contains the '
                      'covariates of interest.')
     },
     description="Build and run linear mixed effects model on balances."
```
Comment (ebolyen) [inline q2_gneiss/regression/_regression.py]: Needs a trailing space

File: q2_gneiss/regression/_regression.py [hunk 2]
```diff
@@ -69,7 +87,7 @@ def lme_regression(output_dir: str,
     name='Simplicial Linear mixed effects regression',
     input_descriptions={
         'table': ('The feature table containing the samples in which '
-                  'simplicial regression with mixed effects will be performed'
+                  'simplicial regression with mixed effect '
                   'will be performed.'),
         'tree': ('A hierarchy of feature identifiers where each tip'
```
Comment (ebolyen) [inline q2_gneiss/regression/_regression.py]: looks like this one needs a trailing space as well

File: q2_gneiss/regression/_regression.py [hunk 3]
```diff
@@ -78,9 +96,15 @@ def lme_regression(output_dir: str,
                  'present in this tree.')
     },
     parameter_descriptions={
-        'formula': 'Statistical formula specifying the statistical model.',
+        'formula': 'Statistical formula specifying the statistical model.'
```
Comment (ebolyen) [inline q2_gneiss/regression/_regression.py]: Add a trailing space

**Review by ebolyen — CHANGES_REQUESTED** [review]
Just some issues with string concatenation

**Review by ebolyen — APPROVED** [review]

**Comment by mortonjt** [comment]
Yay

**Comment by ebolyen** [comment]
Thanks @lisa55asil and @mortonjt!

**Pull request merged**